Tools/TestWebKitAPI/Tests/WebCore/ApplicationManifestParser.cpp

```diff
@@ -36,6 +36,18 @@
 using namespace WebCore;
 
 namespace WebCore {
+static inline std::ostream& operator<<(std::ostream& os, const ApplicationManifest::Direction& direction)
+{
+    switch (direction) {
+    case ApplicationManifest::Direction::Auto:
+        return os << "ApplicationManifest::Direction::Auto";
+    case ApplicationManifest::Direction::Ltr:
+        return os << "ApplicationManifest::Direction::Ltr";
```

Comment:
Nit:
```suggestion
        return os << "ApplicationManifest::Direction::LTR";
```

Reply:
If I do that, the linter complains I should either do `Ltr` or `kLTR`. What’s the right thing to do?

Reply:
Ah, if the linter complains... the linter wins :)